src/base/bittorrent/resumedatastorage.cpp

```diff
@@ -33,6 +33,7 @@
 #include <QMetaObject>
 #include <QMutexLocker>
 #include <QThread>
+#include <QVector>
```

Comment:
Shouldn't this be

```cpp
#include <QList>
```

?

Reply:
No. `QList` is included in `resumedatastorage.h`.